src/scripts/main.js

```diff
@@ -1,3 +1,14 @@
 'use strict';
 
-// write your code here
+const table = document.querySelector('table');
+const tableRows = table.rows;
+
+for (const row of tableRows) {
+  const columnPosition = row.cells[1];
+  const columnPositionCopy = columnPosition.cloneNode(true);
+  const columnSalary = row.cells[4];
+
+  columnSalary.remove();
+  row.append(columnPositionCopy);
+  row.append(columnSalary);
```

Comment:
you can just paste it
```suggestion
  row.lastElementChild.before(columnPositionCopy);
```